giskard/utils/display.py

```diff
@@ -9,3 +9,10 @@ def format_number(value, n=3):
         return f"{value:.{n}e}"
 
     return value
+
+
+def truncate(text, max_length=240, ellipsis="…"):
+    """Truncates a text to the given length, adding an ellipsis at the end if needed."""
+    if len(text) > max_length:
+        return text[: max_length - len(ellipsis)] + ellipsis
+    return text
```

Comment:
Potential issues?
- multi-byte characters 
- `len(ellipsis) > max_length`
- `max_length < 0`

Reply:
I wouldn’t really care, for something like this which is for internal usage I assume responsible usage. I put the default to 240 chars because I don’t know how the servers handles multi-byte, in principle it’s a varchar(255) but who knows the encoding…

Reply:
ok